src/pages/explore.tsx

```diff
@@ -151,6 +160,138 @@ const KaldbQueryRenderer = ({ model }: SceneComponentProps<KaldbQuery>) => {
   );
 };
 
+const KalDBFieldsRenderer = ({ model }: SceneComponentProps<FieldStats>) => {
+  // TODO: Loading state
+  // const { timeseriesLoading, logsLoading } = model.useState();
+  const { fields, topTenMostPopularFields } = model.useState();
+
+  const getIcon = (field: Field): string => {
```

Comment:
We probably want to support icons for text fields, number fields, date fields, and boolean if we want to match Kibana - the raw types can be seen here - https://github.com/slackhq/kaldb/blob/master/kaldb/src/main/java/com/slack/kaldb/logstore/opensearch/OpenSearchAdapter.java#L203